external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaStreamingWriteSupport.scala

```diff
@@ -41,10 +41,10 @@ case object KafkaWriterCommitMessage extends WriterCommitMessage
  * @param schema The schema of the input data.
  */
 class KafkaStreamingWriteSupport(
-    topic: Option[String], producerParams: Map[String, String], schema: StructType)
+    topic: Option[String], producerParams: ju.Map[String, Object], schema: StructType)
```

Comment:
I know it was there before but it's a good chance to fix the style (one argument per line)

Reply:
Fixed.